manifest.yml

```diff
@@ -1,12 +1,12 @@
 manifest_version: 1
-name: ImageNormalizer
-kind: crafter
-type: pod
-description: '`ImageNormalizer` resizes and normalizes images'
+name: RankingEvaluator
+kind: evaluator
+type: executor
+description: '`RankingEvaluator` exposes different type of evaluation metrics'
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
 version: 0.0.1
 license: apache-2.0
```

Comment:
check out the keywords at https://github.com/jina-ai/executor-template/blob/main/.github/GUIDELINES.md#how-to-choose-keywords

There are quite a few fields dropped in the Hubble https://github.com/jina-ai/executor-template/blob/main/.github/GUIDELINES.md#how-to-write-manifestyml